cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -207,7 +212,7 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
           -- version that this node will declare that it understands, when it
           -- is in the Babbage era. Since Babbage is currently the last known
           -- protocol version then this is also the Babbage protocol version.
-          Praos.babbageProtVer = ProtVer 7 1,
+          Praos.babbageProtVer = ProtVer 7 0,
```

Comment:
Why the bump back to 7.0?

Reply:
because this is a hack (see the comment above) that is only needed up until the moment of the hard fork. after the hard fork, I think it is better that the header reflect the truth.

Reply:
"this" meaning "putting a protocol version in the block header that we don't actually intended to transition to"